ovs/extensions/storageserver/storagedriver.py

```diff
@@ -371,9 +371,9 @@ def save(self, client=None, force_reload=False):
                 with remote(client.ip, [LocalStorageRouterClient]) as rem:
                     changes = copy.deepcopy(rem.LocalStorageRouterClient(self.remote_path).update_configuration(self.remote_path))
             reloaded = True
-        except Exception as ex:
-            if 'ClusterNotReachableException' not in str(ex):
-                raise
+        except (ClusterNotReachableException, RuntimeError) as exception:
```

Comment:
This extra checking is done quite often in this PR.
Isn't it better to let a function decide if it was a ClusterNotReachble/RuntimeError with the message?